### ISSUE TITLE
Selected rendering seems weird

### ISSUE THREAD

**Issue by ThebestkillerTBK** [MeteorDevelopment/meteor-client#3041]
### Describe the bug

![image](https://user-images.githubusercontent.com/62250232/206885546-8ccb1461-2a62-4350-967c-7589faec4943.png)
![image](https://user-images.githubusercontent.com/62250232/206892985-f7dba528-2fec-41ee-9e77-ec5e8f6e1a7c.png)

### Steps to reproduce

```bash
go to search
the text glitched(img1)

if you re-expand the setting the situation goes worse(img2)
```

### Meteor Version

0.5.2

### Minecraft Version

1.19.3

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by I-Got-You-Dead** [comment]
When was this even added, I just updated and it was there

**Comment by MineGame159** [comment]
Fixed in 7f3ca30571336d3f43e997833eceb91d66d54036.
